pandas/tools/plotting.py

```diff
@@ -359,7 +359,10 @@ def scatter_matrix(frame, alpha=0.5, figsize=None, ax=None, grid=False,
     density_kwds = density_kwds or {}
 
     # workaround because `c='b'` is hardcoded in matplotlibs scatter method
-    kwds.setdefault('c', plt.rcParams['patch.facecolor'])
+    if 'color' in kwds:
```

Comment:
Ahh gotcha. Thanks!
So one of the problems turned out that difference between `c` and `color` was an edge border on the markers. We can take care of that by setting the default to `'none'`. After making the changes here is the current behavior:
![image](https://cloud.githubusercontent.com/assets/15242567/21254074/c93c54f6-c317-11e6-9587-bc2e9d10bf06.png)
![image](https://cloud.githubusercontent.com/assets/15242567/21254080/d0d495ac-c317-11e6-819d-0a9091355157.png)
![image](https://cloud.githubusercontent.com/assets/15242567/21254083/d618af76-c317-11e6-9de2-3d98dbd887cd.png)
All are now the same.